packages/backup-restore-operator/charts/templates/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1 
+kind: Deployment
+metadata:
+  name: backup-restore-operator
+  namespace: {{ .Release.Namespace }}
+  labels:
+    resources.cattle.io/operator: backup-restore
+spec:
+  selector:
+    matchLabels:
+      resources.cattle.io/operator: backup-restore
+  template:
+    metadata:
+      labels:
+        resources.cattle.io/operator: backup-restore
+    spec:
+      serviceAccountName: backup-restore-operator-serviceaccount
```

Comment:
Nit: can this name be `backup-restore-operator`?

Reply:
Would be nice if we can use `{{ .Chart.Name }}` to avoid hard-coding it